src/pages/javascript/standard-objects/array/array-prototype-map/index.md

```diff
@@ -3,11 +3,18 @@ title: Array.prototype.map
 ---
 ## Array.prototype.map
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/standard-objects/array/array-prototype-map/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+The `.map()` method loops through the given array and executes a provided function on each of the element. It returns a new array which contains the the results of the function calls on each the elements.
```

Comment:
>The `.map()` method loops through the given array and executes ~a~ provided function on each ~of the~ element. It returns a new array which contains ~the~ the results of the function call~s~ on each ~the~ element~s~.

The `.map()` method loops through the given array and executes the provided function on each element. It returns a new array which contains the results of the function call on each element.